src/ardupilot/ArduPilotProtocol.h

```diff
@@ -0,0 +1,96 @@
+#include "../network/websocket/WebSocketProtocol.h"
+#include "../network/websocket/WebSocketServer.h"
+#include "../world_interface/data.h"
+
+#include <condition_variable>
+#include <mutex>
+
+using json = nlohmann::json;
```

Comment:
`using` declarations in header files is bad because it results in namespace pollution. In header files everything should be referred to by the fully-qualified name